tsconfig.json

```diff
@@ -23,17 +11,22 @@
     "rootDir": "./src",
     "strict": true,
     "moduleResolution": "node",
-    "typeRoots": [
-      "src",
-      "node_modules/@types",
-      "types"
-    ],
-    "types": [
-      "node"
-    ],
```

Comment:
Allow TS to resolve the types automatically when included in `node_modules/@types`.
We just need to make sure our package types are in the final tarball before releasing.